uftrace.c

```diff
@@ -630,6 +630,7 @@ static int parse_option(struct opts *opts, int key, char *arg)
 		break;
 
 	case 'U':
+		opts->unpatch = opt_add_string(opts->unpatch, arg);
 		opts->patch = opt_add_prefix_string(opts->patch, "!", arg);
```

Comment:
This is redundant and I think we can use opts->patch (I know it's annoying).  You can ignore ones without the '!' prefix later.